tokio-test/Cargo.toml

```diff
@@ -24,5 +24,10 @@ publish = false
 [dependencies]
 assertive = { git = "http://github.com/carllerche/assertive" }
 pin-convert = "0.1.0"
-tokio-timer = { version = "0.3.0", path = "../tokio-timer" }
 tokio-executor = { version = "0.2.0", path = "../tokio-executor" }
+tokio-io = { version = "0.2.0", path = "../tokio-io" }
+tokio-sync = { version = "0.2.0", path = "../tokio-sync" }
+tokio-timer = { version = "0.3.0", path = "../tokio-timer" }
+
+[dev-dependencies]
+tokio = { path = "../tokio" }
```

Comment:
I wasn't sure about if this is desirable, but it made writing the `io.rs` tests easier...

Reply:
:+1: I think we will eventually need to set `version = "*"` though when releasing.